s64da_benchmark_toolkit/prepare.py

```diff
@@ -62,7 +62,7 @@ def supports_cluster(self):
         if not version:
             return False
 
-        if version.major <= 4 and version.minor < 1:
+        if version.major < 4 or version.major <= 4 and version.minor < 1:
```

Comment:
Now you have overlapping conditions, which makes it hard to read. The `<=` should really be a ` ==`, because only if the major versions are identical you want to compare the minor ones. This does not change the result, though.